### PR TITLE
New update README.md

### PR DESCRIPTION
**Pull request by fkaminski**
New update README.md with further Information ...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,16 +2,23 @@
...
 
+The details below are only necessary if you want to install the Docs locally.\
+If you just want to contribute content, read the [CONTRIBUTING](CONTRIBUTING.md).
 
 ## Cloning
 
-The project installs the Hugo Learn theme as a git submodule. Thus when cloning 
-the repository, you need to use the `--recurse-submodules` parameter:
+The project installs the Hugo Learn theme as a git submodule.\
+Thus when cloning the repository, you need to use the `--recurse-submodules` parameter:
```
Comment (fritzmg) [inline README.md]: Why this change?

File: README.md [hunk 2]
```diff
@@ -2,16 +2,23 @@
 
 The documentation for the Contao project will be maintained in this repository.
 
+The details below are only necessary if you want to install the Docs locally.\
```
Comment (fritzmg) [inline README.md]: "Docs" -> "docs"

File: README.md [hunk 3]
```diff
@@ -21,7 +28,6 @@ To update the theme after cloning, simply run the following command:
 git submodule foreach git pull origin master
 ```
 
-
```
Comment (fritzmg) [inline README.md]: There should be two empty lines before each headline.

File: README.md [hunk 4]
```diff
@@ -55,6 +61,15 @@ make clean
 
 Cleans the build directory.
 
+## Note
+
+Your are not dependent on the [MAKEFILE](Makefile).\
```
Comment (fritzmg) [inline README.md]: "MAKEFILE" -> "Makefile"

File: README.md [hunk 5]
```diff
@@ -55,6 +61,15 @@ make clean
...
+For Example: Windows User can also execute the hugo CLI commands manually like:
```
Comment (fritzmg) [inline README.md]: "Example" -> "example"!

"User" -> "user"

Also a windows user can still use `make` as well.

**Review by fritzmg — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (fkaminski) [reply README.md]: Hi fritzmg,
i have edit your suggestions on my fork: https://github.com/fkaminski/docs/blob/master/README.md

What are the further steps now?

Comment (fritzmg) [reply README.md]: I think you did your edits in your master branch - however, this PR is from your `patch-2` branch ;)

**Comment by fkaminski** [comment]
Ok, restored the patch-2 branch in my fork and made the corrections in this branch ...

File: README.md [hunk 6]
```diff
@@ -2,16 +2,21 @@
...
 The documentation for the Contao project will be maintained in this repository.
 
+The details below are only necessary if you want to install the docs locally.\
+If you just want to contribute content, read the [CONTRIBUTING](CONTRIBUTING.md).
 
 ## Cloning
 
-The project installs the Hugo Learn theme as a git submodule. Thus when cloning 
-the repository, you need to use the `--recurse-submodules` parameter:
+The project installs the Hugo Learn theme as a git submodule. Thus when cloning the repository, you need to use the `--recurse-submodules` parameter:
```
Comment (fritzmg) [inline README.md]: Again, why this change? ;)

Line breaks should happen at or after 80 characters. Since you did not change the content, you shouldn't change anything here.

File: README.md [hunk 7]
```diff
@@ -2,16 +2,21 @@
 
 The documentation for the Contao project will be maintained in this repository.
 
+The details below are only necessary if you want to install the docs locally.\
```
Comment (fritzmg) [inline README.md]: The `\` seems unnecessary here.

File: README.md [hunk 8]
```diff
@@ -55,6 +59,15 @@ make clean
 
 Cleans the build directory.
 
+## Note
+
+Your are not dependent on the [Makefile](Makefile).\
+For example: Windows user can also execute the hugo CLI commands manually like:
```
Comment (fritzmg) [inline README.md]: Why the line break and the `\`? Convention is to make a line break at or after 80 characters (not breaking a word).

Also: "user" -> "users".

And still: this is not really related to the operating system. Windows users can use `make` as well, when installed. It's just that `make` is available with most Unix setups.

**Review by fritzmg — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (fkaminski) [reply README.md]: Hm - i thought that this is re(implement) in my patch-2 branch:
https://github.com/fkaminski/docs/tree/patch-2

File: README.md (hunk 7 above)
Comment (fkaminski) [reply README.md]: Ok Fixed ...

File: README.md (hunk 8 above)
Comment (fkaminski) [reply README.md]: Ok, tried to improve this context ...
What do you think?

File: README.md (hunk 6 above)
Comment (fritzmg) [reply README.md]: No, you have completely removed the line break after "cloning" now. Just restore the original paragraph.

Comment (fkaminski) [reply README.md]: Ok - that way ...

**Review by fritzmg — APPROVED** [review]

**Comment by fritzmg** [comment]
Thank you  @fkaminski
